examples/api/lib/widgets/custom_multi_child_layout/custom_multi_child_layout.0.dart

```diff
@@ -0,0 +1,100 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for CustomMultiChildLayout
+
+import 'package:flutter/material.dart';
+
+void main() => runApp(const MyApp());
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  static const String _title = 'Flutter Code Sample';
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: _title,
+      home: Scaffold(
+        appBar: AppBar(title: const Text(_title)),
+        body: const ExampleWidget(),
+      ),
+    );
+  }
+}
+
+class _SnakeLayoutDelegate extends MultiChildLayoutDelegate {
+  _SnakeLayoutDelegate(
+      {required this.ids, required this.numCols, required this.rowHeight});
```

Comment:
```suggestion
  _SnakeLayoutDelegate({
    required this.ids,
    required this.numCols,
    required this.rowHeight,
  });
```
This should be the more common style in the flutter repo.